rocAL/rocAL_pybind/amd/rocal/pipeline.py

```diff
@@ -112,7 +113,7 @@ def __init__(self, batch_size=-1, num_threads=-1, device_id=-1, seed=-1,
         self._check_ops = ["CropMirrorNormalize"]
         self._check_crop_ops = ["Resize"]
         self._check_ops_decoder = ["ImageDecoder", "ImageDecoderSlice" , "ImageDecoderRandomCrop", "ImageDecoderRaw"]
-        self._check_ops_reader = ["FileReader", "TFRecordReaderClassification", "TFRecordReaderDetection",
+        self._check_ops_reader = ["labelReader", "TFRecordReaderClassification", "TFRecordReaderDetection",
```

Comment:
why is this changed? FileReader seems correct

Reply:
I thought so too. [Here](https://github.com/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/blob/master/rocAL/rocAL_pybind/amd/rocal/readers.py#L45) it takes it as labelReader, and I do not see a FileReader

Reply:
Confirmed with @swetha097, it's the name that's there